### PR TITLE
Base Url in RequestBuilder

### PR DESCRIPTION
**Pull request by markiyanshvets**

### CODE REVIEW

File: Api/BccPay.Core.Infrastructure/PaymentProviders/RequestBuilders/Implementations/BaseRequestBuilder.cs [hunk 1]
```diff
@@ -1,16 +1,20 @@
-﻿using BccPay.Core.Shared.Helpers;
+﻿using System.Text.RegularExpressions;
+using BccPay.Core.Shared.Helpers;
 
 namespace BccPay.Core.Infrastructure.PaymentProviders.RequestBuilders.Implementations
 {
     internal class BaseRequestBuilder
     {
-        protected string GetRedirectUrl(bool isMobile, string forMobile, string forBrowser, string paymentId = null, string attemptId = null)
+        protected string GetRedirectUrl(bool isMobile, string forMobile, string forBrowser, string paymentId = null,
```
Comment (yuriy-butkevych) [inline Api/BccPay.Core.Infrastructure/PaymentProviders/RequestBuilders/Implementations/BaseRequestBuilder.cs:8]: @markiyanshvets we can now remove `forMobile` URL as we are not sending headers from UI anymore

File: Api/BccPay.Core.Shared/Helpers/HttpContextHelper.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public static class HttpContextHelper
+    {
+        private static IHttpContextAccessor _httpContextAccessor;
+
+        private static HttpContext Current 
+            => _httpContextAccessor.HttpContext;
+
+        public static string AppBaseUrl =>
+            $"{Current.Request.Scheme}://{Current.Request.Host}{Current.Request.PathBase}";
```
Comment (yuriy-butkevych) [inline Api/BccPay.Core.Shared/Helpers/HttpContextHelper.cs:13]: Host in our case is `app-samvirk-payments.azurewebsites.net`, but we need a referrer

File: Api/BccPay.Core.Infrastructure/PaymentProviders/RequestBuilders/Implementations/BaseRequestBuilder.cs (hunk 1 above)
Comment (markiyanshvets) [reply Api/BccPay.Core.Infrastructure/PaymentProviders/RequestBuilders/Implementations/BaseRequestBuilder.cs]: done

**Review by yuriy-butkevych — CHANGES_REQUESTED** [review]

File: Api/BccPay.Core.Shared/Helpers/HttpContextHelper.cs (hunk 2 above)
Comment (markiyanshvets) [reply Api/BccPay.Core.Shared/Helpers/HttpContextHelper.cs]: done

**Review by yuriy-butkevych — APPROVED** [review]

**Pull request merged**